### PR TITLE
Avoid NPE if generating a license without an expiration date.

### PR DESCRIPTION
**Pull request by andreidan**
## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: enterprise/licensing/src/main/java/io/crate/license/LicenseTool.java [hunk 1]
```diff
@@ -57,10 +57,17 @@ public static void main(String[] args) throws Exception {
         try {
             OptionSet optionSet = parser.parse(args);
             keyPath = keyArg.value(optionSet);
-            expirationDate = DataTypes.TIMESTAMP.value(expirationDateArg.value(optionSet));
+            String expirationDateArgValue = expirationDateArg.value(optionSet);
+            if(expirationDateArgValue != null) {
```
Comment (seut) [inline enterprise/licensing/src/main/java/io/crate/license/LicenseTool.java]: minor: missing space

**Comment by seut** [comment]
> @seut I saw this PR in jenkins indicating we could use a blank expiration date - crate/jenkins-dsl@028775c#diff-f3299ef8d63773cd3828d9415b54b87aR11

Right, could be blank as well. But would it make much more sense to make it optional? In general i mean, no matter what the jenkins task does.

**Comment by andreidan** [comment]
> > @seut I saw this PR in jenkins indicating we could use a blank expiration date - [crate/jenkins-dsl@028775c](https://github.com/crate/jenkins-dsl/commit/028775c)#diff-f3299ef8d63773cd3828d9415b54b87aR11
> 
> Right, could be blank as well. But would it make much more sense to make it optional? In general i mean, no matter what the jenkins task does.

Ah, an optional arg, yes.

**Comment by andreidan** [comment]
Closing this as the expiration date is not optional. Leaving it as mandatory arg.

**Pull request closed without merging**